src/ApparentHorizons/StrahlkorperDataBox.hpp

```diff
@@ -0,0 +1,208 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "ApparentHorizons/Strahlkorper.hpp"
+#include "DataStructures/DataBox.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// Holds tags and ComputeItems associated with a Strahlkorper.
+namespace StrahlkorperDB {
+
+// This struct supplies shorter names for longer types used below.
+template <typename Frame>
+struct types {
+  using ThetaPhi = tnsr::i<DataVector, 2, ::Frame::Spherical<Frame>>;
+  using OneForm = tnsr::i<DataVector, 3, Frame>;
+  using Vector = tnsr::I<DataVector, 3, Frame>;
+  using Jacobian =
+      Tensor<DataVector, tmpl::integral_list<std::int32_t, 2, 1>,
+             index_list<SpatialIndex<3, UpLo::Up, Frame>,
+                        SpatialIndex<2, UpLo::Lo, ::Frame::Spherical<Frame>>>>;
+  using InvJacobian =
+      Tensor<DataVector, tmpl::integral_list<std::int32_t, 2, 1>,
+             index_list<SpatialIndex<2, UpLo::Up, ::Frame::Spherical<Frame>>,
+                        SpatialIndex<3, UpLo::Lo, Frame>>>;
+  using InvHessian =
+      Tensor<DataVector, tmpl::integral_list<std::int32_t, 3, 2, 1>,
+             index_list<SpatialIndex<2, UpLo::Up, ::Frame::Spherical<Frame>>,
+                        SpatialIndex<3, UpLo::Lo, Frame>,
+                        SpatialIndex<3, UpLo::Lo, Frame>>>;
+  using SecondDeriv = tnsr::ii<DataVector, 3, Frame>;
+};
+
+template <typename Frame>
+struct Strahlkorper : db::DataBoxTag {
```

Comment:
Do you want this to be documented?

Reply:
ping (have something that links to `::Strahlkorper`)